lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -188,6 +188,18 @@ export default TestModule.extend({
       }
       hook.apply(module, Array.prototype.slice.call(arguments, 1));
     };
+    context.register = function() {
```

Comment:
IMHO, these belong in the main `test-module` file, as they are not really related to components...